src/libs/actions/Policy.js

```diff
@@ -737,6 +737,14 @@ function generatePolicyID() {
     return _.times(16, () => Math.floor(Math.random() * 16).toString(16)).join('').toUpperCase();
 }
 
+function openWorkspaceMembersPage(policyID, clientPolicyMembers) {
+    if (!policyID) {
+        return;
+    }
+
```

Comment:
Hmm when would this to be falsey?

Reply:
This was falsey when the policy didn't exist locally in Onyx, but I realized passing `route.params.policyID` is better since this will never be falsey